lib/generators/comfy/blog/blog_generator.rb

```diff
@@ -16,14 +16,27 @@ def self.next_migration_number(dirname)
       end
 
       def generate_migration
-        destination   = File.expand_path("db/migrate/01_create_blog.rb", destination_root)
-        migration_dir = File.dirname(destination)
-        destination   = self.class.migration_exists?(migration_dir, "create_blog")
-
-        if destination
-          puts "\e[0m\e[31mFound existing create_blog migration. Remove it if you want to regenerate.\e[0m"
-        else
-          migration_template "db/migrate/01_create_blog.rb", "db/migrate/create_blog.rb"
+        migrations = %w[
+          01_create_blog
+          02_add_file_reference_to_blog_posts
+        ]
+        version_regex = %r{\d.*?_}
+
+        migrations.each do |identifier|
+          destination = File.expand_path(
+            "db/migrate/#{identifier}.rb", destination_root
+          )
+          migration_dir = File.dirname(destination)
+          destination   = self.class.migration_exists?(
+            migration_dir, identifier.gsub(version_regex, "")
+          )
+
+          if destination
+            puts "\e[0m\e[31mFound existing #{identifier.gsub(version_regex, '')} migration. Remove to regenerate.\e[0m"
```

Comment:
I was struggling with the best way to control line-length here with rubocop. Seems `\` in double quoted strings wanted to add white space but it didn't like `+` concatenation. I settled for reducing the text, but I have no strong opinions here.